auth-web/src/components/auth/manage-business/EntityManagement.vue

```diff
@@ -271,6 +258,28 @@
         </template>
       </ModalDialog>
 
+      <!-- Business Unavailable Dialog for unaffiliated business-->
+      <ModalDialog
+        class="business-unavailable-dialog"
```

Comment:
Is this class (and dialog-class on line 267) used for anything?

(I saw you use the ref in the unit tests.)

Reply:
> It's to add a new css style for dialog. It's used for unaffiliated modals only (refers to uxpin https://preview.uxpin.com/a86dccdd20ac62828f965c5eea10f81b828a9853#/pages/163299967/simulate/sitemap).

> #### The new dialog:
> ![image](https://user-images.githubusercontent.com/60866283/259193929-f64d6ece-f2aa-4fd9-b0f9-3e3ac24fd13f.png)
> 
> The wording is slightly different according to different NR types

It's for unaffiliated modals. Please see comments and screenshot above.